src/App/Components/DisplayArticle/Ratings.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable react/no-unused-state */
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { ToastContainer, toast } from 'react-toastify';
+import propTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import Rater from 'react-star-ratings';
+import Rating from './Rating';
+import actions from '../../../Redux/Actions/articleRatings';
+
+const { getRatings, rate } = actions;
+
+export class Ratings extends Component {
+  static propTypes = {
+    getRatings: propTypes.func,
+    ratings: propTypes.object,
+    errorMessage: propTypes.string,
+    successMessage: propTypes.string,
+    rate: propTypes.func
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      average: undefined,
+      allUsers: undefined,
+      stats: [],
+      rating: 0,
+      redirect: false,
+      areRatingsShown: false,
+      isLoggedIn: false
+    };
+  }
+
+  componentDidMount() {
+    const { getRatings: getArticleRatings } = this.props;
+    getArticleRatings('What-is-a-Version-1-UUID');
```

Comment:
@nkpremices I think you have to get `slug` from the article you are reading not adding it manually like this.
or maybe if there is a reason behind this, you may explain more about this.

Reply:
This component will be added on @MCFrank16's component.
I will be taking the slug from there

Reply:
@Ramadhan0